src/helper/signToken.js

```diff
@@ -0,0 +1,20 @@
+import jwt from 'jsonwebtoken';
+import config from '../config/config';
+
+export const signToken = (data) => {
+  const { JWTKEY } = config;
+  const {
+    email, password
+  } = data;
+  try {
+    return jwt.sign(
+      {
+        email, password
+      },
+      JWTKEY,
+      { expiresIn: '1d' }
+    );
+  } catch (error) {
+    throw new Error('No token');
+  }
+};
```

Comment:
why this file since you don't have anywhere you are using it?

Reply:
I will have to delete it.